manim/scene/scene.py

```diff
@@ -946,6 +1032,7 @@ def play_internal(self, skip_rendering=False):
             named parameters affecting what was passed in ``args``,
             e.g. ``run_time``, ``lag_ratio`` and so on.
         """
+        self.sync_play_running = True
```

Comment:
Is there a reason to set assign this variable here? Will I have to call wait for the async to work?